### PR TITLE
Add mqtt broker

### LINKED ISSUE #47: Establish MQTT broker for Flotilla
**Describe the new feature you would like to see**
The Flotilla application should have an associated MQTT broker/server which is responsible for streaming data from the different ISAR applications and robots on the facility to Flotilla. In addition, the broker should bridge to a fleet management broker to enable streaming at scale.

**Describe the solution you'd like**
This issue aims to get started with the MQTT broker for the Flotilla application. The bare minimum is adding an MQTT broker (https://www.hivemq.com/) which runs through a docker container and may be started together with the Flotilla application through docker compose. 

**How will this feature affect the current Threat Model?**
The first iteration will only include a broker which is local to the Flotilla system and as such no data streaming out of the internal network.
- Authentication to the broker must be handled for all applications which should subscribe/publish
- Communication security must be handled when bridging to a fleet management broker (future task)

### PR DESCRIPTION
**Pull request by aestene**
Closes #47

### CODE REVIEW

File: broker/mosquitto/config/mosquitto.conf [hunk 1]
```diff
@@ -0,0 +1,906 @@
...
+# for config files. All files that end in '.conf' will be loaded as
+# a configuration file. It is best to have this as the last option
+# in the main file. This option will only be processed from the main
+# configuration file. The directory specified must not contain the
+# main configuration file.
+# Files within include_dir will be loaded sorted in case-sensitive
+# alphabetical order, with capital letters ordered first. If this option is
+# given multiple times, all of the files from the first instance will be
+# processed before the next instance. See the man page for examples.
+#include_dir
```
Comment (oysand) [inline broker/mosquitto/config/mosquitto.conf:906]: Newline

File: broker/mosquitto/config/passwd_file [hunk 2]
```diff
@@ -0,0 +1 @@
+mosquitto:$6$aB/lZEmj399gcmFN$5F2lAaVaGZ6dDYpvwlv4zONbn2CqwsypZuL0Gy3O5u0SgmzSVUurI7oSO/tTD1vuFaW9NaM6nb6XIJnd65vUEw==
```
Comment (oysand) [inline broker/mosquitto/config/passwd_file:1]: Is this the password?

Comment (aestene) [reply broker/mosquitto/config/passwd_file:1]: This is the hash of the default passoword which has been set to `default`. This way the default application has a password, although a deployed solution would have to change this locally of course.

File: broker/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+```
+docker build -t flotilla-broker .
+```
+
+The broker may be started as
+
+```
+docker run -it -p 1883:1883 -p 9001:9001 flotilla-broker:latest
+```
+
```
Comment (tsundvoll) [inline broker/README.md:16]: Maybe also add an example of how to test the docker image, by subscribing and publishing to a topic?

**Review by vetlek — APPROVED** [review]
LGTM. Maybe squash to one commit?

File: broker/README.md (hunk 3 above)
Comment (tsundvoll) [reply broker/README.md:16]:
```suggestion

docker run -it -p 1883:1883 -p 9001:9001 flotilla-broker:latest

mosquitto_sub -t 'topic' -u mosquitto -P default
mosquitto_pub -t 'topic' -m 'hello world' -u mosquitto -P default
```

Comment (aestene) [reply broker/README.md:16]: Done.

**Pull request merged**